src/utils/rttc.ts

```diff
@@ -37,6 +37,9 @@ const typeOf = (v: Value) => {
 const isNumber = (v: Value) => typeOf(v) === 'number'
 const isString = (v: Value) => typeOf(v) === 'string'
 const isBool = (v: Value) => typeOf(v) === 'boolean'
+const isObject = (v: Value) =>
+  typeOf(v) === 'object' && !(v instanceof Closure) && !(v instanceof Function) && !Array.isArray(v)
```

Comment:
Similarly here, the check for Closure is redundant. I'm not sure if you meant to have null be an object here, but note that null will return true for isObject.